addons/l10n_pe/models/res_district.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class District(models.Model):
+    _name = 'l10n.pe.district'
```

Comment:
l10n_pe.res.city.district please in order to indicate it is child of city and belongs to l10n_pe

Reply:
Do I need to change the name of the district's file in data (csv) too?

Reply:
Done